Sources/MapboxMaps/Foundation/MapboxMap.swift

```diff
@@ -835,6 +841,59 @@ extension MapboxMap {
                                  featureId: featureId,
                                  stateKey: stateKey)
     }
+
+}
+
+// MARK: - View Annotations
+
+extension MapboxMap {
+
+    // TODO: Add documentation
+    public func setViewAnnotationPositionsUpdateListenerFor(listener: ViewAnnotationPositionsListener) {
+        __map.setViewAnnotationPositionsUpdateListenerFor(listener)
+    }
+
+    /**
+     * Add view annotation.
+     *
+     * @return position for all views that need to be updated on the screen or null if views' placement remained the same.
+     */
+    public func addViewAnnotation(forId id: String, options: ViewAnnotationOptions) {
+        __map.addViewAnnotation(forIdentifier: id, options: MapboxCoreMaps.ViewAnnotationOptions(options))
+    }
+
+    /**
+     * Update view annotation if it exists.
+     *
+     * @return position for all views that need to be updated on the screen or null if views' placement remained the same.
+     */
+    public func updateViewAnnotation(forId id: String, options: ViewAnnotationOptions) {
```

Comment:
What happens if no annotation exists with the specified id?

Reply:
The call is simply ignored at the moment.